server/settings.py

```diff
@@ -222,3 +222,6 @@
 AUDIT_EXPIRY_MINUTES = int(env('AUDIT_EXPIRY_MINUTES', 60 * 24 * 7))  # 7d
 PUBLISH_QUEUE_EXPIRY_MINUTES = int(env('PUBLISH_QUEUE_EXPIRY_MINUTES', 60 * 24 * 7))  # 7d
 ARCHIVED_EXPIRY_MINUTES = int(env('ARCHIVED_EXPIRY_MINUTES', 60 * 24 * 60))  # 60d
+
+#disable use of XMP for photo assignments
+PLANNING_USE_XMP_FOR_PIC_ASSIGNMENTS = strtobool(env('PLANNING_USE_XMP_FOR_PIC_ASSIGNMENTS', 'false'))
```

Comment:
you can just set it to `= False` imo we don't want to use different config here on each env like uat/prod but keep it same everywhere

Reply:
ok, will do